build/index.md

```diff
@@ -27,3 +27,63 @@ BuildKit is the backend evolution from the Legacy Builder, it comes with new
 and much improved functionality that can be powerful tools for improving your
 builds' performance or reusability of your Dockerfiles, and it also introduces
 support for complex scenarios.
+
+
```

Comment:
```suggestion

```

Reply:
Removed additional empty lines which do not render.

Reply:
Does not look solved to me, still two new lines.

Reply:
@dockertopia ^